### PR TITLE
Add BigQueryEmulatorContainer

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+public class BigQueryEmulatorContainerTest {
+
+    @Test
+    public void test() throws Exception {
+        try (
+            BigQueryEmulatorContainer container = new BigQueryEmulatorContainer("ghcr.io/goccy/bigquery-emulator:0.4.3")
+        ) {
+            container.start();
+
+            String url = String.format("http://%s", container.getEmulatorHttpEndpoint());
```
Comment (bsideup) [inline modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java:25]: Just curious - WDYT about adding `getEmulatorURL()` to make it easier to work with?

File: modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            BigQueryOptions options = BigQueryOptions
+                .newBuilder()
+                .setProjectId("test")
```
Comment (bsideup) [inline modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java:28]: We probably need `container.getProjectId()`, as "test" is a magical constant here

Comment (eddumelendez) [reply modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java:28]: Yep, I think we need to add this for all our gcloud TC implementations 😬

File: modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java (hunk 1 above)
Comment (eddumelendez) [reply modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java:25]: Agree! I was just being "consistent" with the other gcloud TC implementations 🙈

**Pull request merged**